src/utils/documentProducer/variables.ts

```diff
@@ -158,3 +158,18 @@ export const missingRequiredFields = (
     return false;
   });
 };
+
+export const getDependingVariablesStableIdsFromOtherDeliverable = (variablesWithValues: VariableWithValues[]) => {
+  const existingStableIds = new Set<number>();
```

Comment:
nit: in theory our stableIDs are in a string column. In practice they are all numbers in the spreadsheet. This is likely fine for now but we could in theory put strings in the spreadsheet.

Reply:
Oh, maybe I should update that then